app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -442,30 +436,40 @@ class _ReplyContentWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (msg is TextMessage) {
-      return messageWidget!;
-    } else if (msg is ImageMessage) {
+    if (msg is ImageMessage) {
+      var imageMsg = msg as ImageMessage;
       return Padding(
         padding: const EdgeInsets.all(8.0),
-        child: ConstrainedBox(
-          constraints: const BoxConstraints(maxHeight: 100, maxWidth: 125),
-          child: ClipRRect(
-            borderRadius: BorderRadius.circular(6.33),
-            child: Image.memory(
-              base64Decode(msg?.metadata?['base64']),
-              fit: BoxFit.fill,
-              cacheWidth: 125,
-            ),
-          ),
+        child: ImageMessageBuilder(
```

Comment:
nitpick: but this isn't a builder, it's a widget, isn't it?

Reply:
Yes, its a widget. Named it for the simplicity as its custom image renderer for```Chat``` widget.
<img width="561" alt="Screenshot 2023-08-07 at 2 28 57 PM" src="https://github.com/acterglobal/a3/assets/68579938/7ab5736e-4a6e-41b0-a045-b5e5a96405cf">